Screens/ConfirmEmail.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { StyleSheet, Text, View, ScrollView, Image, StatusBar, FlatList, TouchableOpacity, TouchableNativeFeedback,
+  Button,
+  Platform } from 'react-native';
+import { DefaultTheme, Headline, Paragraph, TextInput,
+  Appbar, AppbarAction, AppbarBackAction, AppbarContent, AppbarHeader, 
+  Card, CardContent,
+  // Button,
+  ListAccordion, Divider,
+  ListSection, withTheme,
+  Toolbar, ToolbarBackAction, ToolbarContent, ToolbarAction, 
+  FAB, DrawerSection, Colors,
+  Provider as PaperProvider } from 'react-native-paper';
+import { createStackNavigator, DrawerActions } from 'react-navigation';
+import { Constants } from 'expo';
+import { RectangleButton, RoundButton, WideImage } from '../Components';
+import Slideshow from 'react-native-slideshow';
+
+export default class ConfirmEmail extends React.Component {
+    static navigationOptions = ({ navigation }) => {
+      return {
+          headerBackImage: require('../assets/back-icon.svg'),
+          headerTitle:'Sign In',
+          headerLeft:(<Text style={styles.headerLeft}>X</Text>)
+        };
```

Comment:
This can be a straight up object. No need to be a function. More readable as just an object.